src/lib/components/Cytoscape.react.js

```diff
@@ -676,6 +764,14 @@ Cytoscape.propTypes = {
          */
         boundingBox: PropTypes.object,
     }),
+    /**
+ * Define a custom context menu
+ */
+    contextMenuData: PropTypes.array,
+    /**
+     * Retrieve relevant data when context menu item is clicked
+     */
+    tapContextMenu: PropTypes.object,
```

Comment:
Can we be more precise with these types?
```
contextMenuData: PropTypes.arrayOf(
    PropTypes.exact({
        id: PropTypes.string.isRequired,
        content: PropTypes.string.isRequired,
        ...
    })
)
```

Reply:
Actually maybe they want different names - mostly the event-type props get names `<something>Data` and the config props don't. So maybe `contextMenuData` -> `contextMenu`, and `tapContextMenu` -> `contextMenuData`?